arcgis-ios-sdk-samples/Content Display Logic/Controllers/CategoryTableViewController.swift

```diff
@@ -47,15 +52,29 @@ class ContentTableViewController: UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // initialize download progress view
+        // Initialize download progress view.
         let downloadProgressView = DownloadProgressView()
         downloadProgressView.delegate = self
         self.downloadProgressView = downloadProgressView
+        
+        dataSource = UITableViewDiffableDataSource(tableView: tableView, cellProvider: { [unowned self] tableView, indexPath, _ in
+            let sample = self.displayedSamples[indexPath.row]
+            
+            let cell = tableView.dequeueReusableCell(withIdentifier: "ContentTableCell", for: indexPath) as! ContentTableCell
+            cell.titleLabel.text = sample.name
+            cell.detailLabel.text = sample.description
+            cell.isExpanded = expandedRowIndexPaths.contains(indexPath)
+            return cell
+        })
+        updateDataSource()
```

Comment:
For next reviewer's context: the table view won't reload if the sample's favorite state is changed when the table view is presented (like an iPad split view). We decided that favorite state-change notification is an overkill here.